test/spoom/sorbet/metrics_test.rb

```diff
@@ -33,10 +33,6 @@ def test_parses_metrics
            ]
           }
         ERR
-        assert_equal("MyRepo", metrics.repo)
```

Comment:
I let the data lines 18 to 34 as to show that it doesn't break if the output actually contains those fields.